utils/aws_health_utils.py

```
@@ -71,3 +71,64 @@ def get_all_events():
+        # Send the email
+        response = ses_client.send_email(
+            Source=sender_email,
+            Destination={'ToAddresses': [recipient_email]},
+            Message={
+                'Subject': {'Data': subject},
+                'Body': {'Text': {'Data': body}}
+            }
+        )
+        print(f"Email sent! Message ID: {response['MessageId']}")
+    except Exception as e:
+        raise ValueError(f"Failed to send email:",e)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)